docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+
+services:
+  new_fastapi_db:
+    image: postgres:16
+    container_name: new_fastapi_db_container
+    env_file:
+      - .env
```

Comment:
please add the required environment variables to the readme file or use a default value in the compose file

Reply:
Done